recipes-kernel/linux/kernel-devsrc.bbappend

```diff
@@ -0,0 +1,27 @@
+do_install_prepend() {
+    cd ${S}
+    if [ ${LINUX_GIT_BRANCH} = "linux-5.10.y-cip" ]; then
```

Comment:
It's not good to rely on a particular branch name.
Can we write it like this?
`if ${@bb.utils.vercmp_string_op(d.getVar("KERNEL_VERSION"), '5.10', '>=')}; then`

I think that a valid version will be available for the KERNEL_VERSION because it is used after do_configure().
https://www.yoctoproject.org/docs/2.3/ref-manual/ref-manual.html#var-KERNEL_VERSION